JXHTTP/JXHTTPOperation.h

```diff
@@ -303,4 +303,15 @@ typedef NSURLRequest * (^JXHTTPRedirectBlock)(JXHTTPOperation *operation, NSURLR
  */
 + (void)setNetworkActivityIndicatorManager:(id <JXNetworkActivityIndicatorManager>)networkActivityIndicatorManager;
 
+#if __IPHONE_OS_VERSION_MIN_REQUIRED >= __IPHONE_2_0
```

Comment:
Is this necessary? The podspec specifies that the deployment target is 5.0. There are a few places in this repo that use `__IPHONE_OS_VERSION_MIN_REQUIRED`, but it doesn't seem like it's necessary.